ionizer/ops.py

```diff
@@ -62,7 +67,15 @@ def compute_matrix(phi):  # pylint: disable=arguments-differ
         array([[0.        +0.j        , 0.95533649-0.29552021j],
                [0.95533649+0.29552021j, 0.        +0.j        ]])
         """
-        return qml.math.stack([[0, qml.math.exp(-1j * phi)], [qml.math.exp(1j * phi), 0]])
+        if qml.math.get_interface(phi) == "tensorflow":
+                phi = qml.math.cast_like(phi, 1j)
+
+        a = (0 + 0j)
+        b = qml.math.exp((0 - 1j) * phi)
+        c = qml.math.exp((0 + 1j) * phi)
```

Comment:
Instead of defining `c`, can use `qml.math.conj` in the matrix construction.

Reply:
Fixed.